### ISSUE TITLE
Name it electron-site-launcher or something more general

### ISSUE THREAD

**Issue by clemens-tolboom** [K00sKlust/slither.io-bot-manager#3]
@K00sKlust darn :-p about the name slither.io-bot-manager ... your app can launch any user script on any web site and probe for statistics. That's why I named it **electron-site-launcher**

Maybe leave out electron as that doesn't matter.

(I need to check I can launch it on a different site first)

**Comment by K00sKlust** [comment]
I agree that the name should be changed, but I think **electron-site-launcher** is too general. I would like to highlight the ability of the application to inject scripts and monitor the instances..

**Comment by clemens-tolboom** [comment]
> I would like to highlight the ability of the application to inject scripts and monitor the instances.

Agreed
